Source/PeleLMeX_BCfill.H

```diff
@@ -77,7 +87,8 @@ struct PeleLMCCFillExtDirState
           }
           // bcnormal() is defined in pelelmex_prob.H in problem directory in
           // /Exec
-          bcnormal(x, m_nAux, s_ext, idir, 1, time, geom, *lprobparm, lpmfdata);
+          T::bcnormal(
+            x, s_in, s_ext, idir, 1, time, geom, *lprobparm, lpmfdata);
```

Comment:
I think the CodeQL warning here is spurious due to NUM_SPECIES varying from case to case

Reply:
That's my guess too, it might not have been a problem before bcnormal got static.